openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -860,7 +855,7 @@ def create_force(self, system, topology, **kwargs):
                     force.addTorsion(atom_indices[1], p[0], p[1], p[2],
                                      improper_periodicity, improper_phase, improper_k)
         logger.info(
-            '{} impropers added, each applied in a six-fold trefoil'.format(
+            '{} impropers added, each applied in a three-fold trefoil'.format(
```

Comment:
Leaving this link as a pointer to the six vs. three-fold improper conversation: https://github.com/openforcefield/openforcefield/pull/105#issuecomment-379431703